Plugins.md

```diff
@@ -30,6 +30,37 @@ to enable it this can be done via clicking on the green check mark
 
 ![thold plugin enable](images/cacti_thold_enable.JPG)
 
+### Updating a Plugin
+
+Updating a plugin not only keep you up to date with the latest release of the code for that plugin
+it also allows you to keep all of your settings for that plugin ensure you DO NOT UNINSTALL the plugin
+you must instead disable the plugin which will not drop the SQL tables and thus save your settings
+
+- Disable the plugin you want to update
+
+- backup the current plugin folder typically in cacti|_path/plugins/plugin_name
+
+```console
+mv /var/www/html/cacti/plugin/thold /tmp
+```
+- git clone or download the new version of the plugin
+
+```console
+git clone https://github.com/Cacti/plugin_thold.git
+```
```

Comment:
Once you have cloned a repo, you do not need to repeat the move and reclone.  You can simply use `git pull` to pull down the latest version, then `git checkout <tag>` to move to a specific version.

Reply:
I generally do a clone, in production environments.